contrib/node/examples/src/java/org/pantsbuild/testproject/jsresources/BUILD

```diff
@@ -0,0 +1,17 @@
+jvm_binary(name = 'jsresources',
```

Comment:
This has the same name as the directory, so no need to specify the name (it's the "default" target).